stable/license/README.md

```diff
@@ -0,0 +1,126 @@
+# Install NuoDB License on Kubernetes
+
+When using the Helm package manager to deploy NuoDB, a license is not installed by default.
+
+To view the details such as date of expiry (`expires`), name of the license holder (`holder`), and the type of license (`type`) , run:
+
+```
+  nuocmd get effective-license
+```
+
+For example, if the *Enterprise License* is installed:
+
+```
+nuocmd --show-json get effective-license
+
+{
+  "decodedLicense": {
+    "expires": "2025-01-31T15:33:40.465134",
+    "holder": "3DS Solutions",
+    "type": "ENTERPRISE"
+  },
+  "effectiveForDomain": true,
+  "encodedLicense":
+    "-----BEGIN LICENSE-----
+    <base64-encoded data>
+    -----END LICENSE-----"
+}
+```
+To install a license or to upgrade an existing license, choose one of the following options:
+
+* Redeploy the Admin Processes (APs) using Helm [RECOMMENDED]
+* Use the `nuocmd set license` command
+
+## Redeploy the Admin Processes (APs) using Helm
+
+### To install an *Enterprise License* or a *Limited License* for a new Admin domain
+
+1. Contact <nuodb.support@3ds.com> to obtain a new Limited License key or an Enterprise License key.
+
+2. Make a copy of the admin chart’s `values.yaml` file.
+
+   ```
+   helm show values nuodb/admin > admin-values.yaml
+   ```
+
+3. Edit the `admin-values.yaml` file.
+
+   Replace the contents of `configFiles:` with the contents of the license file. For example:
+
+   ```
+   configFiles:
+    nuodb.lic: |-
+    #   "PUT YOUR BASE64 ENCODED LICENSE CONTENT HERE"
+    -----BEGIN LICENSE-----
+    <base64-encoded data>
+    -----END LICENSE-----
+   ```
+
+   > **NOTE**:
+   > Paste the entire license file, including the lines `-----BEGIN LICENSE-----` and `-----END LICENSE-----`.
+
+4. Save the changes to the `admin-values.yaml` file.
+
+5. Re-install the admin chart specifying `--values admin-values.yaml`.
```

Comment:
This says re-install, but the heading of this subsection says "new Admin domain", which is confusing. Should this just say "Install"?